src/components/composite/alert.js

```diff
@@ -6,10 +6,10 @@ import CreateBloks from "../../utilities/createBloks";
 import "../../styles/alert.css";
 
 const Alert = ({ blok }) => {
-  const isLinkDark = blok.type === "warning";
-  console.log(isLinkDark, "isLinkDark");
+  const { type, alertCta, label, heading, body } = blok;
```

Comment:
Restructure here, because we need `blok` in SbEditable component

Reply:
You can actually do this all in one go and still retain the blok parent. 
```suggestion
const Alert = ({ blok: { type, alertCta, label, heading, body }, blok}) => {
```